client/blocks/import/capture/capture-input.tsx

```diff
@@ -69,13 +72,124 @@ const CaptureInput: FunctionComponent< Props > = ( props ) => {
 		}
 	}
 
+	function isIDN( url: string ) {
```

Comment:
Can we have this function outside the component body?
Each render is recreating the function.

Reply:
Thanks, I've handled it here